uber/templates/base.html

```diff
@@ -224,8 +224,8 @@
 {% block header %}{% endblock %}
 {% block content %}{% endblock %}
 
-<div id="bottomAnchor" /></div> <!-- end mainContainer -->
-</div>
+<div id="bottomAnchor"></div> <!-- end mainContainer -->
+
 {% block scripts %}
```

Comment:
@thaeli need your eyes on this, it appeared to be working fine in my browser but the HTML5 validator that helped me fix the other error was complaining loudly that ```<div id="bottomAnchor" />``` was going to be ignored and there was an extra </div> in here.  this one's not super-critical for the stripe fix, just, throwing it in since I was in here anyway.